code/modules/psionics/psionic_items/psi_temp_items.dm

```diff
@@ -34,6 +34,52 @@
 		qdel(src)
 		return
 
+/obj/item/flame/pyrokinetic_spark
+	name = "pyrokinetic flame"
+	desc = "A flickering psionic induced flame."
+	icon = 'icons/obj/psionic/occpyrospark.dmi'
+	icon_state = "pyrospark"
+	item_state = "pyrospark"
+	var/burntime = 120
+	w_class = ITEM_SIZE_HUGE
+	slot_flags = null
+	attack_verb = list("burnt", "singed")
+	lit = 1
+	var/mob/living/carbon/holder
+
+/obj/item/flame/pyrokinetic_spark/Process()
+	if(isliving(loc))
+		var/mob/living/M = loc
+		M.IgniteMob()
+	var/turf/location = get_turf(src)
+	burntime--
+	if(burntime < 1)
+		burn_out()
+		return
+	if(location)
+		location.hotspot_expose(700, 5)
+		return
+
+/obj/item/flame/pyrokinetic_spark/proc/burn_out()
+		visible_message("The [src.name] flickers away from your hand into nothingness.")
```

Comment:
```suggestion
		visible_message("The [src.name] flickers away from your hand like it was nothing.")
```

Reply:
in general this messages are the same as for the other items so a bit of variety of flavour would be fine